### PR TITLE
Improve CLI further, update ListCommand to accommodate for empty list and add more tests in LogicMangerTest

### PR DESCRIPTION
**Pull request by e1121208**

### CODE REVIEW

File: src/main/resources/view/DarkTheme.css [hunk 1]
```diff
@@ -1,237 +1,180 @@
...
-    --hover-bg-color: rgba(255, 215, 0, 0.4);
-    --border-radius: 12px;
-    --font-family-regular: 'Raleway', sans-serif;
-    --font-family-header: 'Lora', serif;
-    --transition-speed: 0.3s;
-    --shadow-effect: drop-shadow(3px 3px 6px rgba(0, 0, 0, 0.7));
-    --hover-shadow: drop-shadow(4px 4px 10px rgba(255, 215, 0, 0.7));
-    --focused-border: 3px solid rgba(255, 215, 0, 0.8);
-}
-
```
Comment (muhdbhz) [inline src/main/resources/view/DarkTheme.css:16]: Can you comment this out instead love <3

**Review by muhdbhz — CHANGES_REQUESTED** [review]
Pls see below

File: src/main/resources/view/DarkTheme.css (hunk 1 above)
Comment (e1121208) [reply src/main/resources/view/DarkTheme.css]: Considerations has been considered

**Review by muhdbhz — APPROVED** [review]
LGTM!

**Pull request merged**